test/test.js

```diff
@@ -42,6 +44,54 @@ describe('mercurius', function() {
       .end(done);
   });
 
+  it('successfully registers additional machine', function(done) {
+    request(mercurius.app)
+      .post('/register')
+      .send({
+        token: tokenToUnregister,
+        machineId: 'machine2',
+        endpoint: 'endpoint',
+        key: 'key',
+      })
+      .expect(function(res) {
+        assert.equal(res.status, 200);
+        assert.equal(res.text, tokenToUnregister);
+      })
+      .end(done);
+  });
+
+  it('returns 404 if bad token provided', function(done) {
+    request(mercurius.app)
+      .post('/register')
+      .send({
+        token: 'notexisting',
+        machineId: 'machine of a not existing token',
+        endpoint: 'endpoint',
+        key: 'key',
+      })
+      .expect(404, done);
+  });
+
+  it('successfully unregisters machines', function(done) {
+    request(mercurius.app)
+      .post('/unregisterMachine')
+      .send({
+        token: tokenToUnregister,
+        machineId: 'machine',
+      })
+      .expect(200, done);
+  });
+
+  it('replies with 404 when trying to unregister a non registered machine', function(done) {
+    request(mercurius.app)
+      .post('/unregisterMachine')
+      .send({
+        token: tokenToUnregister,
+        machineId: 'non-existing-machine',
+      })
+      .expect(404, done);
+  });
+
```

Comment:
We should also test that notify sends the notification to all machines in a token set.